testing/regress/ecl/exampleplugin.ecl

```diff
@@ -0,0 +1,28 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+//class=embedded
```

Comment:
@AttilaVamos These were both on the redis example tests but could you give comments for each such that I can understand them better and perhaps add them to this example, please? E.g. the use of 'embedded' is perhaps misleading and the need for the '3rdparty', does this just prevent certain building and testing until you can manually set stuff up or is it all automated?